lint/private/lint_aspect.bzl

```diff
@@ -46,33 +46,75 @@ def should_visit(rule, allow_kinds, allow_filegroup_tags = []):
 
 _OUTFILE_FORMAT = "{label}.{mnemonic}.{suffix}"
 
-# buildifier: disable=function-docstring
-def report_files(mnemonic, target, ctx):
-    report = ctx.actions.declare_file(_OUTFILE_FORMAT.format(label = target.label.name, mnemonic = mnemonic, suffix = "report"))
-    outs = [report]
+def output_files(mnemonic, target, ctx):
+    """Declare linter output files.
+
+    Args:
+        mnemonic: used as part of the filename
+        target: the target being visited by a linter aspect
+        ctx: the aspect context
+
+    Returns:
+        tuple of struct() of output files, and the OutputGroupInfo provider that the rule should return
+    """
+    human_out = ctx.actions.declare_file(_OUTFILE_FORMAT.format(label = target.label.name, mnemonic = mnemonic, suffix = "txt"))
```

Comment:
maybe its just be but I'm allergic to `.txt` extensions from Windows days of old. I suggest `human.report`, `human.exit_code`, `machine.report`, `machine.exit_code` suffixes so its clear form the file names what they are and how they match up to the output groups

Reply:
Also if we used `.human.report` it would break anyone on current Aspect CLI as we just scan for `endswith(report)`

Reply:
renamed `.txt` to `.out`